(SP: 0.5) Fixed price overlapping the bookmark

docia opened the pull request:
- [x]  Fixed price overlapping the bookmark

### Before:
<img width="1205" alt="before_overlapping" src="https://github.com/user-attachments/assets/90302dd3-7ce1-4132-a2f8-a988b35df51d">

### After:
https://github.com/user-attachments/assets/04cab8bd-afd2-4c9b-8872-8014ca81d9e1

docia committed:
fixed price overlapping bookmarks

PavloDolia reviewed (commented):
How about changing styles.bookmarkButton.right from -20px to -13px? I think it would look 
a little nicer
<img width="1186" alt="Screenshot 2024-09-04 at 23 58 22" src="https://github.com/user-attachments/assets/199dcfe3-699b-4ac5-80b6-2f56413b5ecd">
<img width="741" alt="Screenshot 2024-09-04 at 23 58 50" src="https://github.com/user-attachments/assets/16236dfa-e6c0-4a10-8996-b36b80a4747e">

docia commented:
> How about changing styles.bookmarkButton.right from -20px to -13px? I think it would look a little nicer <img alt="Screenshot 2024-09-04 at 23 58 22" width="1186" src="https://private-user-images.githubusercontent.com/123400767/364545760-199dcfe3-699b-4ac5-80b6-2f56413b5ecd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1MzY5NzksIm5iZiI6MTcyNTUzNjY3OSwicGF0aCI6Ii8xMjM0MDA3NjcvMzY0NTQ1NzYwLTE5OWRjZmUzLTY5OWItNGFjNS04MGI2LTJmNTY0MTNiNWVjZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwNVQxMTQ0MzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02YmY0OWQyMTJlMDU3YTFmNGYzYjE0NGE1N2JjNzNhOTdhMmJhMTZmOTZhZmM0MGRiNDAwNTJkOWVkMjAwOTA2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.hBGWnaIwAqTWtaE8zbhdn17K1L7_odT5kqcjO1G9FGc"> <img alt="Screenshot 2024-09-04 at 23 58 50" width="741" src="https://private-user-images.githubusercontent.com/123400767/364545764-16236dfa-e6c0-4a10-8996-b36b80a4747e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1MzY5NzksIm5iZiI6MTcyNTUzNjY3OSwicGF0aCI6Ii8xMjM0MDA3NjcvMzY0NTQ1NzY0LTE2MjM2ZGZhLWU2YzAtNGExMC04OTk2LWIzNmI4MGE0NzQ3ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwNVQxMTQ0MzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01Yzg3NWQxNGYxMTc0MWE5NzI1ODc2YWQ5NjIyNzA1NWFjMzVmNGQyN2FmOGY2N2ZmNjkwOTU3MmI1NzgyNmFlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.sIVwGHCt0gxhrOaXDs2-VFQcD5YSHE7xSL0ymsVV2lQ">

@PavloDolia I like your suggestion! However, I changed to `-12px` as I learned a long time ago that using **values that are multiples of two** in CSS provides better compatibility with high-resolution screens, simplifies responsive design and typography, and promotes more precise and harmonious display of UI elements.

docia committed:
update styles

OliaHrytsak commented:
Now he bookmark position is not according to mockup
I suggest to limit user in the prise per course or another decision - cut the prise 

![image](https://github.com/user-attachments/assets/baaa3557-c593-4493-9de1-d01fbed65da6)

docia committed:
updated styles

docia commented:
> Now he bookmark position is not according to mockup I suggest to limit user in the prise per course or another decision - cut the prise
> 
> ![image](https://private-user-images.githubusercontent.com/127191734/364813729-baaa3557-c593-4493-9de1-d01fbed65da6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYxNDQwMzEsIm5iZiI6MTcyNjE0MzczMSwicGF0aCI6Ii8xMjcxOTE3MzQvMzY0ODEzNzI5LWJhYWEzNTU3LWM1OTMtNDQ5My05ZGUxLWQwMWZiZWQ2NWRhNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxMlQxMjIyMTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02OWE4YzAxMmMyOGU0Nzk4NTUyY2M0MmVjZWExMGUwYmU3YjA0NGRjN2Y5ZGY1YmU2NzYxZTRlMTdhM2QzOTI1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.bYP4bqr0_hsU9d-Z5n-bEDtt1fImCBMtVOVSxTMuODI)

thank you for your suggestion. I've been thinking about it but eventually decided that limiting the ability to set a particular price would not be right. people have different assessment criteria for choosing a price, so I believe it is right to leave them an opportunity to set any prices they want. 
I changed some pixels in the last commit and updated the video in the PR. All elements, including the bookmark, are aligned in one line, which looks better now, as for me.
